core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -96,23 +98,15 @@ object ConsoleProducer {
     props.put("serializer.class", config.valueEncoderClass)
     props.put("send.buffer.bytes", config.socketBuffer.toString)
     props.put("topic.metadata.refresh.interval.ms", config.metadataExpiryMs.toString)
-    props.put("client.id", "console-producer")
-
-    props
-  }
 
-  private def producerProps(config: ProducerConfig): Properties = {
```

Comment:
Hmm, so the refactoring here seems like it's going the opposite direction we'd want -- now there's unlabeled logic in main() that's really about the config and we've duplicated code for getting the user-specified configs included in both the old and new producer configs.

Reply:
@ewencp no news since August on this :(